lusSTR/workflows/strs.smk

```diff
@@ -19,7 +19,7 @@ separate = config["separate"]
 def get_sample_IDs(input, uas, output, software, separate):
     convert_out = f"{output}.txt"
     format_out = f"{output}.csv"
-    if software == "efm" and separate is False:
+    if (software == "efm" or software == "mpsproto") and separate is False:
```

Comment:
Another way of implementing this conditional is `if software in ("efm", "mpsproto"):`. This is more concise, but I'm not sure it's clearer; in any case, this is more of an FYI and not a suggestion for this PR.